.github/workflows/continuous-integration-workflow.yml

```diff
@@ -1,11 +1,40 @@
 name: Continous Integration Tests
-on:
-  pull_request:
-    branches:
-      - master
+on: [push, pull_request]
 jobs:
-  test:
-    name: ODS resources setup and provisioning tests
+  webhook-proxy:
+    name: Webhook Proxy tests
+    runs-on: ubuntu-18.04
```

Comment:
As a question... which version should we adopt 18.04 or 16.04? I used 16.04 just because it was the default for TravisCI

Reply:
Why stick with an older version? For me this should be latest as much as possible. I'm only avoid using `latest` because then the actual version is undefined.

Reply:
I agree... should we change also the other test then to 18.04? Just to make it consistent

Reply:
Or create a issue/PR for it maybe

Reply:
👍